### PR TITLE
Make linter happy for 08-wasm

### PR DESCRIPTION
**Pull request by DimitrisJim**
## Description

pleases the linter

closes: #XXXX

### Commit Message / Changelog Entry

```bash
chore: if anything, this PR was definitely a chore
```

see the [guidelines](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) for commit messages. (view raw markdown for examples)

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#pull-request-targeting)).
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/main/docs/docs/building-modules/11-structure.md) and [Go style guide](../docs/dev/go-style-guide.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/main/testing/README.md#ibc-testing-package).
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`).
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Provide a [commit message](https://github.com/cosmos/ibc-go/blob/main/docs/dev/pull-requests.md#commit-messages) to be used for the changelog entry in the PR description for review.
- [ ] Re-reviewed `Files changed` in the Github PR explorer.
- [ ] Review `Codecov Report` in the comment section below once CI passes.

### CODE REVIEW

File: modules/light-clients/08-wasm/client/cli/tx.go [hunk 1]
```diff
@@ -31,7 +31,7 @@ func newStoreCodeCmd() *cobra.Command {
 
 			fileName := args[0]
 
-			code, err := ioutil.ReadFile(fileName)
+			code, err := os.ReadFile(fileName)
```
Comment (damiannolan) [inline modules/light-clients/08-wasm/client/cli/tx.go:34]: 👍

File: modules/light-clients/08-wasm/keeper/keeper_test.go [hunk 2]
```diff
@@ -3,7 +3,7 @@ package keeper_test
 import (
 	"testing"
 
-	"github.com/stretchr/testify/suite"
+	testifysuite "github.com/stretchr/testify/suite"
```
Comment (damiannolan) [inline modules/light-clients/08-wasm/keeper/keeper_test.go:6]: correct me if I'm wrong, but was this aliased before `testsuite` someplace else or am I mistaken?

File: modules/light-clients/08-wasm/types/types_test.go [hunk 3]
```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/stretchr/testify/suite"
+	testifysuite "github.com/stretchr/testify/suite"
```
Comment (damiannolan) [inline modules/light-clients/08-wasm/types/types_test.go:9]: ditto on alias

**Review by crodriguezvega — APPROVED** [review]
Looks good to me!

**Review by damiannolan — APPROVED** [review]
I assume tests are expected to be failing at the moment.

Nice clean up tho! LGTM

File: modules/light-clients/08-wasm/keeper/keeper_test.go (hunk 2 above)
Comment (DimitrisJim) [reply modules/light-clients/08-wasm/keeper/keeper_test.go:6]: yea, slightly confused me too tbh, `testsuite` is another package in the e2e module. In that big linter PR, we went with `testifysuite` to try to differentiate from that one too.

Comment (damiannolan) [reply modules/light-clients/08-wasm/keeper/keeper_test.go:6]: okay sounds good to me!

**Pull request merged**